plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -537,7 +537,10 @@ codemodernizer.notification.info.transformation_stop.title=Transformation stoppe
 codemodernizer.notification.info.transformation_summary.content=We need to download the transformation summary, please try again after this has completed.
 codemodernizer.notification.info.transformation_summary.title=Transformation summary download started!
 codemodernizer.notification.info.view_troubleshooting_guide=View troubleshooting guide
-codemodernizer.notification.warn.invalid_project.description.reason.invalid_jdk_versions=We could not find an upgrade eligible application. We currently support upgrade of Maven Java applications of version 8 and 11.
+codemodernizer.notification.info.view_troubleshooting_guide.download_diff.url=https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/troubleshooting-code-transformation.html#w24aac14c20c19c11
+codemodernizer.notification.info.view_troubleshooting_guide.maven_issues.url=https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/troubleshooting-code-transformation.html#w24aac14c20c19b7
+codemodernizer.notification.info.view_troubleshooting_guide.prerequisites.url=https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html#prerequisites
```

Comment:
These links could live in the utils file and be declared as const variables?

Reply:
Agreed that these strings are not to be transformed/localized. Please move out of this file